src/containers/TagManager.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+import Page from './Page';
+import ContentHeader from '../components/ContentHeader';
+import InPlaceEditor from '../components/forms/InPlaceEditor';
+import Tagger from '../components/forms/Tagger';
+
+import settings from '../settings';
+
+require('../../css/react-tag-input.css');
+
+@connect(state => {
+  return state.pipelines;
+})
+@Radium
+export default class TagManager extends React.Component {
+
+  componentWillMount() {
+    this.setState({ tags: [ 'top_commenter', 'banned', 'moderator', 'martian', 'ufo' ] });
+  }
+
+  confirmDeletion(index) {
+    var tagsCopy = this.state.tags;
+    tagsCopy.splice(index, 1);
+    this.setState({ tags: tagsCopy });
+  }
+
+  tagAdderChangeHandler(event) {
+    this.setState({ adderValue: event.target.value });
+  }
+
+  tagAdderClickHandler() {
+    if (this.state.adderValue) {
+      this.setState({ tags: [ ...this.state.tags, this.state.adderValue ] });
+      React.findDOMNode(this.refs.tagAdderInput).value = ''; 
```

Comment:
with React 0.14, they moved DOM manipulation stuff to a separate module, `ReactDOM` and deprecated the old way. so `React.findDOMNode` => `ReactDOM.findDOMNode` [findDOMNode](http://facebook.github.io/react/docs/top-level-api.html#reactdom.finddomnode)